packages/client-recommendation/package.json

```diff
@@ -17,7 +17,7 @@
   ],
   "dependencies": {
     "@algolia/client-common": "4.9.3",
-    "@algolia/requester-common": "4.9.3",
-    "@algolia/transporter": "4.9.3"
+    "@algolia/client-personalization": "4.9.3",
+    "@algolia/requester-common": "4.9.3"
```

Comment:
```suggestion
```

I don't think this is still used here. Can you double-check?

Reply:
It is: https://github.com/algolia/algoliasearch-client-javascript/blob/95cf1089c7af9dbc2772c74d89357ce2d1367608/packages/client-recommendation/src/methods/setPersonalizationStrategy.ts#L1

Reply:
ah yes good call! I was looking in the wrong files